components/arm/fake/fake.go

```diff
@@ -85,34 +85,40 @@ func init() {
 
 // NewArm returns a new fake arm.
 func NewArm(cfg config.Component, logger golog.Logger) (arm.LocalArm, error) {
+	armModel := cfg.ConvertedAttributes.(*AttrConfig).ArmModel
+	modelPath := cfg.ConvertedAttributes.(*AttrConfig).ModelFilePath
+	model, err := buildModel(armModel, modelPath, cfg.Name)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Arm{
+		Name:   cfg.Name,
+		joints: &pb.JointPositions{Values: make([]float64, len(model.DoF()))},
+		model:  model,
+		logger: logger,
+	}, nil
+}
+
+func buildModel(armModel, modelPath, name string) (referenceframe.Model, error) {
```

Comment:
this can just take in the whole config as an argument right?

Reply:
Might as well, also lets you update `buildModel` without changing the signature if we decide `Fake` arms should have more attributes later.

Reply:
great callout :)